app/src/main/java/com/antosik/benchproject/app/common/constants/Constants.kt

```diff
@@ -0,0 +1,8 @@
+package com.antosik.benchproject.app.common.constants
+
+internal class Constants {
+
+    companion object {
```

Comment:
Instead of creating class and then object in class you can just create object 😄

Reply:
Ok, I understand. I deleted it (the whole file) and I created an object in `MoreViewModel.kt`. 😄